Yehor/reactNetflix/src/components/sort-panel/SortPanel.tsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import "./SortPanel.css";
+import Button from "../button";
+import { parse } from "query-string";
+import * as History from "history";
+
+interface ISortPanelProps {
+  moviesLength: number;
+  handlerSortChange(sortBy: string): void;
+  location: History.Location;
+}
+interface ISortPanelState {
+  sortBy: SortBy;
+}
+enum SortBy {
+  Date = "release date",
+  Rating = "rating",
+}
+
+class SortPanel extends React.Component<ISortPanelProps, ISortPanelState> {
+  state = {
+    sortBy: SortBy.Date,
+  };
+
+  sortByDate = () => {
+    this.setState({
+      sortBy: SortBy.Date,
+    });
+    this.props.handlerSortChange("release date");
```

Comment:
this.props.handlerSortChange("release date"); => this.props.handlerSortChange(SortBy.Rating);